shark-log/src/main/java/shark/SharkLog.kt

```diff
@@ -13,45 +13,31 @@ object SharkLog {
     /**
      * Logs a debug message formatted with the passed in arguments.
      */
-    fun d(
-      message: String,
-      vararg args: Any?
-    )
+    fun d(message: String)
 
     /**
      * Logs a [Throwable] and debug message formatted with the passed in arguments.
      */
-    fun d(
-      throwable: Throwable,
-      message: String,
-      vararg args: Any?
-    )
+    fun e(throwable: Throwable, message: String)
```

Comment:
The name is was mirrored after Log.d, ie the logging level (d, e, i, ...) . What we log is different from what level we log it at, and here there's no difference in level so I would stick to "d". It's more of a helper logger to log the stacktrace